subprojects/platform-jvm/src/main/java/org/gradle/jvm/toolchain/JavaInstallationMetadata.java

```diff
@@ -77,4 +77,10 @@ public interface JavaInstallationMetadata {
      */
     @Internal
     Directory getInstallationPath();
+
+    /**
+     * Returns true if this installation corresponds to the currently running JVM.
+     */
+    @Internal
+    boolean isCurrentJvm();
```

Comment:
This change is moved to the separate PR: https://github.com/gradle/gradle/pull/22142